crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/prosecution_concluded/impl/ProsecutionConcludedImpl.java

```diff
@@ -41,33 +35,29 @@ public void execute(ConcludedDTO concludedDTO) {
             courtDataAPIService
                     .updateCrownCourtOutcome(
                             UpdateCCOutcome.builder()
-                                    .repId(maatId)
+                                    .repId(repOrderDTO.getId())
```

Comment:
If we only require the `MAAT_ID` I don't see the need to pass the entire `RepOrderDTO`

Reply:
Good point, not sure how I missed that!